src/pages/models/lib/cooler.js

```diff
@@ -82,9 +78,7 @@ const handle = new Promise((resolve) => {
   }).catch(() => {
     debug('Initial connection attempt failed')
     debug('Starting Scuttlebutt server')
-
-    server(ssbConfig)
-
+    require('../../../../server')
```

Comment:
This is the only thing I'm unsure about. I think that the hierarchical `require()` pattern makes it easier to understand what's going on, but [obviously] the server shouldn't be nested this deeply. Is it best to conform to the dependency graph pattern for consistency (and add an incentive to refactor), or is it better to make an exception here? I'm learning toward the former, but I'm down to trust your opinion on this.

Either way, this is on the top of my list for a refactor.

Reply:
How about I just move server.js into the same folder as cooler and run it from there? Then its all in the same place for you to refactor later

Reply:
Works great for me, thanks for putting up with the mess. :upside_down_face:

Reply:
No mess. Code base is pretty tame. You've done a great job so far!